mkdocs.yml

```diff
@@ -12,6 +12,10 @@ pages:
     - Home: 'networking/index.md'
     - Overview: 'networking/overview.md'
     - SimpleImpl: 'networking/simpleimpl.md'
+  - Data Storage:
+    - Capabilities: 'datastorage/capabilities.md'
```

Comment:
I'm not fully convinced either, but I think the issue is maybe the name of the category instead. The meaning I wanted was "systems that allow you to attach data to things", to go along with "Networking", but that's the best name I could come up with.